tech-far-hub/src/components/layout.tsx

```diff
@@ -1,51 +1,48 @@
-import * as React from "react"
-import { ReactNode } from 'react'
-import '@trussworks/react-uswds/lib/uswds.css'
-import '@trussworks/react-uswds/lib/index.css'
-import './tfh.scss'
+import * as React from "react";
+import { ReactNode } from "react";
+import "@trussworks/react-uswds/lib/uswds.css";
```

Comment:
Is there a reason why these css file are needed here rather than keeping styles in the tfh.scss? I'm sure there's a reason and I'm just not seeing it! :)

Reply:
For the moment, because I haven't totally decide on my plan for how to manage the CSS tbh --- I'll create a ticket to refactor tho, because you're right, this is a silly way to do it. I just wanted the fastest thing that would work. 


The other bummer is that the gatsby-scss plugin depends on a vulnerable package, and I keep hoping it'll get fixed before I have to decide whether to keep it.